src/main/java/org/xerial/snappy/BitShuffle.java

```diff
@@ -0,0 +1,184 @@
+/*--------------------------------------------------------------------------
+ *  Copyright 2011 Taro L. Saito
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *--------------------------------------------------------------------------*/
+//--------------------------------------
+// snappy-java Project
+//
+// BitShuffle.java
+// Since: 2016/03/31
+//
+// $URL$
+// $Author$
+//--------------------------------------
+package org.xerial.snappy;
+
+import java.io.IOException;
+
+public class BitShuffle
+{
+    static {
+        try {
+            impl = SnappyLoader.loadBitShuffleApi();
```

Comment:
How about the case where users only use the `BitShuffle` interfaces in `snappy-java`?
They have already used other default compressors such as gzip, and they'd possibly like to use the interfaces only.